docs/source/i524/lectures-theory.csv

```diff
@@ -11,11 +11,11 @@ Introduction ,Course Introduction,`Slides <https://drive.google.com/file/d/0Bx_s
       ,Introduction,`Video <https://www.youtube.com/watch?v=VWK7QdThOPQ>`_,0:13:59
       ,Introduction - Real World Big Data,`Video <https://www.youtube.com/watch?v=hjrFaXMU1dg&list=PLLO4AVszo1SNIGlqjEBaQjLvc3lgpeI3M&index=2>`_,0:15:28
       ,Introduction - Basic Trends and Jobs,`Video <https://www.youtube.com/watch?v=u5Y0aKbzync&index=3&list=PLLO4AVszo1SNIGlqjEBaQjLvc3lgpeI3M>`_,0:10:57
-Acess Patterns, Data Access Patterns and Introduction to using HPC-ABDS,`Slides <https://iu.box.com/s/uozssvsy0p9fhh0uv8gldwoves2bl0vv>`_,
+Acess Patterns, Data Access Patterns and Introduction to using HPC-ABDS,`Slides <https://iu.app.box.com/files/0/f/16611433111/1/f_122710787684>`_,
     , A. Introduction to HPC-ABDS Software and Access Patterns,`Video <https://www.youtube.com/watch?v=y2HzXjmRf4Y>`_ -  `Resource 1 <http://grids.ucs.indiana.edu/ptliupages/publications/HPC-ABDSDescribedv2.pdf>`_,0:27:45
     , B. Science Examples (Data Access Patterns),`Video <https://www.youtube.com/watch?v=hYJ4_1A4kEU>`_ -  `Resource 2 <http://hpc-abds.org/kaleidoscope/>`_,0:18:38
     , C. Remaining General Access Patterns,`Video <https://www.youtube.com/watch?v=hsEDKM7Zur4>`_,0:11:26
     , D. Summary of HPC-ABDS Layers 1 - 6,`Video <https://www.youtube.com/watch?v=zVPKiGV0fk0>`_,0:14:32
-    , E. Summary of HPC-ABDS Layers 7 - 13,`Video <https://www.youtube.com/watch?v=LxRwGE4sfVM>`_,0:30:52
+    , E. Summary of HPC-ABDS Layers 7 - 13,`Video <https://youtu.be/6Kj9E38lUzU>`_,0:30:52
```

Comment:
This seems to be the same video as the previous one.
What changed between them?

https://youtu.be/6Kj9E38lUzU
https://www.youtube.com/watch?v=LxRwGE4sfVM